### DISCUSSION TITLE
Dragging audio file into audio editor?  is it possible to make it work like images work with image editors?

### DISCUSSION THREAD

**Discussion by hems** [tauri-apps/tauri#6980] [Q&A] [answered]
Hello, thanks a lot for reading my thread and for your help/advice.

I built an audio-related app many years ago ( using nw.js ) and one of the issues i could not solve was dragging audio files into audio editors directly from the app.

The issue is that the "dragged item" from Chrome won't be recognised by the audio editor as an audio file the same way dragging images into an image editor ( such as Photoshop ) is recognised.

Because of that users who need to drag the files from the app need to first save them into the desktop ( or another folder ) and then drag them to their audio editor.

I'm wondering if anyone here has experience solving this problem using Tauri? 

I'm now creating a new application and would like to have this issue sorted as it's very important for the end-users and I would love to use Tauri. 

Apparently this is solved on Electron ( [more info here](https://github.com/electron/electron/pull/6333) ) so I'm guessing should be possible somehow with Tauri but I'm not sure if that feature is somehow implemented?

Thanks a lot!

**Comment by FabianLars** [comment, marked as answer]
This would be an extension of this feature request: https://github.com/tauri-apps/tauri/issues/2593. This means it's not yet implemented, so unless you want to try to fiddle with the OS apis with their Rust bindings (idk how far that could theoretically can get you without changed in Wry/Tao) Tauri probably is not something for you in this context :)
